src/operations.ts

```diff
@@ -242,6 +243,21 @@ const OPERATIONS: ReadonlyArray<Operation> = [
         selectors: { followedThreadsLink: SELECTOR.followedThreadsLink },
         action: REPLACE_FOLLOWED_THREADS_LINK,
     }),
+    new DependentOperation({
+        description: "adaptive width: remove space for .pushColumn>.fixed if empty",
```

Comment:
När är `.pushColumn > .fixed` tom?

Reply:
För övrigt är det tänkt att `description` ska passa in grammatiskt i `Could not ...`; det gör den ju inte nu.